src/Microsoft.ML.Tokenizers/Model/Tiktoken.cs

```diff
@@ -544,5 +640,18 @@ private static unsafe string GetString(ReadOnlySpan<byte> utf8Bytes)
             }
 #endif
         }
+
+        private static bool StringContainInvalidChars(string text)
+        {
+            for (int i = 0; i < text.Length; i++)
+            {
+                if (text[i] == 0xFFFD)
+                {
+                    return true;
+                }
+            }
+
+            return false;
+        }
```

Comment:
```suggestion
        private static bool StringContainInvalidChars(string text) => text.Contains('\uFFFD');
```

Reply:
This text already removed in recent commits and we are just using string.IndexOf now